gufe/vendor/openff/models/types.py

```diff
@@ -23,7 +23,7 @@ def __getitem__(self, t):
 
 
 if TYPE_CHECKING:
-    FloatQuantity = Quantity
+    FloatQuantity = unit.Quantity
```

Comment:
Having to do this is weird typing magic. It's not a great solution but it'll do for now?